src/bot.rs

```diff
@@ -273,6 +288,14 @@ impl RcBot {
         receiver.then(|x| x.map_err(|_| Error::from(ErrorKind::Channel)))
     }
 
+    pub fn default_cmd(&self) -> impl Stream<Item = (RcBot, objects::Message), Error = Error> {
```

Comment:
I don't really like the name here. Perhaps unknown_cmd gives a better hint what this function is supposed to do and, actually, fits better to new_cmd.